packages/formik/src/utils.ts

```diff
@@ -66,18 +66,24 @@ export function getActiveElement(doc?: Document): Element | null {
 /**
  * Deeply get a value from an object via its path.
  */
-export function getIn(
-  obj: any,
-  key: string | string[],
-  def?: any,
-  p: number = 0
-) {
-  const path = toPath(key);
-  while (obj && p < path.length) {
-    obj = obj[path[p++]];
+ export function getIn(
```

Comment:
Looks like the formatting got messed up here. Is this indented correctly?

Reply:
You're right, prettier doesn't seem to run automatically. I just fixed it, and added more tests.